lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -10,7 +10,7 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
   if (userInformation.enrollments) {
     courseElement = (<div>
       <label className="label-course" htmlFor="course">{gettext('Course Name')}</label>
-      <select className="form-control select-course" id="course" value={userInformation.course_id}>
+      <select className="form-control select-course" id="course" defaultValue={userInformation.course_id}>
```

Comment:
Can we use `selected` attribute of option https://www.w3schools.com/tags/att_option_selected.asp. Don't know about `defaultValue`.

Reply:
@tasawernawaz `defaultValue` is react specific to select a value on initialization.